src/main/java/teammates/logic/core/EmailGenerator.java

```diff
@@ -449,7 +443,7 @@ public EmailWrapper generateSystemErrorEmail(
         if (errorMessage == null) {
             int msgTruncateIndex = stackTrace.indexOf("at");
             if (msgTruncateIndex > 0) {
-                errorMessage = stackTrace.substring(0, msgTruncateIndex);
+                errorMessage = stackTrace.substring(0, msgTruncateIndex).trim();
```

Comment:
This is needed otherwise the line breaks will be included. Caught in one of the tests added.